starter-code/index.html

```diff
@@ -3,40 +3,86 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" type="text/css" href="./css/main.css">
 </head>
 <body>
-Premium Discover Help Download
+  <header>
+<nav>
+   <img src="./images/spotify-logo.png"/>
+    <ul id="menu">
```

Comment:
this should be a class not an ID. and ID should only be used for linking within a page and not for styling